<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COVID-19</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="world_pb">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WORLD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="india_pb">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>INDIA</string>
    </property>
   </widget>
   <widget class="QLabel" name="heading_lbl">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>COVID-19 STATISTICS</string>
    </property>
   </widget>
   <widget class="QLabel" name="india_stats_lbl">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Click below to see the statistics of India:-</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="wold_stats_lbl">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Click below to see the statistics of the World:-</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Resource Files/Covid-19.qrc"/>
 </resources>
 <connections/>
</ui>
